docs/mp/faulttolerance/01_overview.adoc

```diff
@@ -70,3 +148,58 @@ NOTE: There is currently _no support_ to configure these executor properties via
 For a complete set of properties available to configure these executors, see
 link:{executor-config}[ServerThreadPoolSupplier] and
 link:{scheduled-executor-config}[ScheduledThreadPoolSupplier].
+
+== Examples
+
+1. The method  `retryWithFallback` shall be called at most 3 times, first
+call plus 2 retries, with a delay of 400 milliseconds between calls. If none of the
+calls is successful, the `onFailure` method shall be called as a fallback mechanism.
++
+[source,java]
+----
+    @Retry(maxRetries = 2, delay = 400L)
+    @Fallback(fallbackMethod = "onFailure")
+    String retryWithFallback() {
+        // ...
+    }
+----
+
+2. The method `timedCircuitBreaker` defines a _rolling window_ of size 10 and a policy
+to open the circuit  breaker after 4 or more failures occur in that window, and to transition
+back to half open state after 3 consecutive and successful runs. Additionally, it sets an overall timeout
+for the invocation of 1.5 seconds.
++
+[source,java]
+----
+    @Timeout(1500)
+    @CircuitBreaker(requestVolumeThreshold = 10,
+        failureRatio = .4
+        successThreshold = 3)
+    void timedCircuitBreaker() throws InterruptedException {
+        // ...
+    }
+----
+
+3. The method `executeWithQueueAndFallback` defines a bulkhead that will limit the number of
+concurrent calls to a maximum of 2; any additional tasks shall be queued up to a maximum of 10.
+Finally, if an error occurs the `onFailure` method shall be called as a fallback mechanism.
+The `@Asynchronous` annotation is needed to enable queueing of bulkhead tasks.
++
+[source,java]
+----
+    @Asynchronous
+    @Fallback(fallbackMethod = "onFailure")
+    @Bulkhead(value = 2, waitingTaskQueue = 10)
+    CompletableFuture<String> executeWithQueueAndFallback() {
+        // ...
+    }
+----
+
+== Additional Information
+
+For additional information about this API, see the
+link:{fault-tolerance-javadoc}[MicroProfile Fault Tolerance Javadocs].
+
+== Reference
+
+* link:{fault-tolerance-spec-url}}[MicroProfile Fault Tolerance]
```

Comment:
At least as rendered when viewing the file on GitHub, this link is broken. Looks like maybe the double `}` after the URL?